packages/pretty-format/src/plugins/html_element.js

```diff
@@ -155,4 +155,4 @@ const print = (
   return result;
 };
 
-module.exports = ({print, test}: Plugin);
+export default ({print, test}: Plugin);
```

Comment:
By adding both the default and named export we support both ES6 module consumers (like rollup) and CommonJS `require` use several places in Jest source code.